vendor/bat-native-ads/src/bat/ads/internal/search_providers.h

```diff
@@ -41,6 +41,8 @@ const std::vector<SearchProviderInfo> _search_providers = {
       false),
   SearchProviderInfo(
       "Google",
+      // TODO(Moritz Haller): Doesn't capture geography specific domains
```

Comment:
Is this something that needs completing now? Or can we raise a ticket and instead of `TODO(Moritiz Haller):` can we change to `TODO(<URL_FOR_ISSUE>`